next_gen_ui_demo/README.md

```diff
@@ -0,0 +1,3 @@
+A demo version of the [Building next generation UIs in Flutter][] codelab.
+
```

Comment:
Can you quickly define what makes it a demo version (a couple sentences about the buttons and other modifications.